arch/arm/soc/st_stm32/stm32f4/Kconfig.defconfig.series

```diff
@@ -12,47 +12,12 @@ source "arch/arm/soc/st_stm32/stm32f4/Kconfig.defconfig.stm32f4*"
 config SOC_SERIES
 	default stm32f4
 
-config SERIAL
-	def_bool y
-
-if SERIAL
-
-config CONSOLE
-	def_bool y
-
-config UART_CONSOLE
-	def_bool y
-
-config UART_STM32
-	def_bool y
-
-endif #SERIAL
-
-if GPIO
-
-config GPIO_STM32
-	def_bool y
-
-config GPIO_STM32_PORTA
-	def_bool y
-
-config GPIO_STM32_PORTB
-	def_bool y
-
-config GPIO_STM32_PORTC
-	def_bool y
+if GPIO_STM32
 
 config GPIO_STM32_PORTD
-       def_bool y
```

Comment:
Why not adding E and H here?
From quick search they are supported on all available SoCs

Reply:
Fixed.